components/ui-api-layer/internal/gqlschema/schema.graphql

```diff
@@ -462,20 +462,19 @@ type Query {
     apis(environment: String!, serviceName: String, hostname: String): [API!]!
 
     remoteEnvironment(name: String!): RemoteEnvironment
-    remoteEnvironments(environment: String, first: Int, offset: Int): [RemoteEnvironment]!
+    remoteEnvironments(environment: String, first: Int, offset: Int): [RemoteEnvironment!]!
     connectorService(remoteEnvironment: String!): ConnectorService!
 
-    environments(remoteEnvironment: String): [Environment]!
-    deployments(environment: String!, excludeFunctions: Boolean): [Deployment]!
-    resourceQuotas(environment: String!): [ResourceQuota]!
-    resourceQuotasStatus(environment: String!): ResourceQuotasStatus!
+    environments(remoteEnvironment: String): [Environment!]!
+    deployments(environment: String!, excludeFunctions: Boolean): [Deployment!]!
+    resourceQuotas(environment: String!): [ResourceQuota!]!
+    resourceQuotaStatus(environment: String!): ResourceQuotasStatus!
```

Comment:
This is breaking change. Did you verify if it is not used? If it is used then please mark old query as deprecated.

Reply:
Sure, it was my fault during rebase. Already fixed